ruby-training/trainees/alan/test_katas/01/math_maddness/spec/math_maddness_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rspec/autorun'
+
+class Aritmetic_operations
```

Comment:
You have an entire class of repeated code here. You should remove any class code from this file and use the `require` statement to import that class into your spec file. Also, consider renaming the file to `arithmetic_operations_spec.rb` to match your class name and follow conventions.